src/components/Content/Breadcrumb.css

```diff
@@ -6,14 +6,14 @@
 
 .separator::after {
   color: #e8e8e8;
-  font-size: 22px;
+  font-size: 16px;
   content: ' > ';
   user-select: none;
 }
 
 .level {
   color: #9c9c9c;
-  font-size: 20px;
+  font-size: 15px;
```

Comment:
I think we shold still increase the font-size of the breadcrumbs to at least 16-17px, so the user can easily touch them.

Reply:
We can increase the touch-target. I think if it looks ok we should definitely make it smaller to save space.

Reply:
do you really think we need a bigger font-size/touch-target? at least on desktop this shouldn't be a problem, on mobile maybe

Reply:
No it's ok for now :D. Leave it at 15.

Reply:
added 10px on top and bottom of the links, so that the touch-target is better
this could be increased/decreased easily if you want to

Reply:
removed it again, otherwise scrolling would be possible in vertical direction